apps/judicial-system/backend/src/app/formatters/custodyNoticePdf.ts

```diff
@@ -80,14 +81,25 @@ const constructCustodyNoticePdf = (
     )
   }
 
+  addNormalText(doc, 'Lögheimili/dvalarstaður: ', 'Helvetica', true)
```

Comment:
This wasn't here before, but with added defendant information I added this to bring more clarity and consistency to the defendant info